config/gen2_feeding_demo.yaml

```diff
@@ -56,7 +56,7 @@ wheelchair:
 feedingDemo:
   heightAbovePlate: 0.15
   heightAboveFood: 0.08
-  heightIntoFood: 0.085
+  heightIntoFood: 0.08
```

Comment:
FYI (leaving this comment here for future reference): this parameter currently has to be changed based on how much the wheelchair sinks based on who is sitting in it.